### ISSUE TITLE
Allow commenting to be turned off

### ISSUE THREAD

**Issue by MorrisonCole** [MorrisonCole/pr-lint-action#253]
I guess some users will just want the status check and nothing else - it can be annoying to get a tonne of comments!

**Comment by Couto** [comment]
I would add that a better solution was to, either:
- Dont add comments again (it is annoying to have 20 comments stating the same thing)
- Update the same comment, I prefer this solution honestly, as the same comment can be edited after the action is successful.

**Comment by MorrisonCole** [comment]
@Couto yep to be honest didn't give it much thought. I'll have a look at updating the comment / preventing multiple comments when I get around to this!

**Comment by gregorybolkenstijn** [comment]
I would love any of the above mentioned options! 🔥

**Comment by thedmeyer** [comment]
@MorrisonCole Would love to see this! Any updates?

**Comment by gregorybolkenstijn** [comment]
@MorrisonCole My team would love this as well, as we've seen PRs with a seemingly endless stream of success messages.

**Comment by gregorybolkenstijn** [comment]
@MorrisonCole Any chance this PR could be merged? https://github.com/MorrisonCole/pr-lint-action/pull/286

**Comment by MorrisonCole** [comment]
Hey folks - I'm really sorry for having this repo on the backburner for such a long time. I have of course seen all your comments and have set some time aside to address all the open issues this weekend. I'll ping all the threads once the released versions are out!

**Comment by MorrisonCole** [comment]
[v1.7.0](https://github.com/MorrisonCole/pr-lint-action/releases/tag/v1.7.0) is now released and adds the approach suggested by @Couto of updating the existing comment. Since I'd still like to add the option to turn off comments entirely, I'm leaving this issue open for now, but the annoying multiple-comments problem is at least fixed for now :)
